### PR TITLE
Add support for terabyte values in numeric startup options (t, tb, T, TB, tib, TiB, TIB)

### PR DESCRIPTION
**Pull request by jsteemann**
Add support for terabytes values in startup options

https://github.com/arangodb/arangodb/pull/17642

### CODE REVIEW

File: 3.11/administration-configuration.md [hunk 1]
```diff
@@ -192,19 +192,21 @@ for small adjustments, you can put local overrides into a file
 
 It is possible to add suffixes to numeric options that will cause ArangoDB to
 multiply the value by a certain factor. This can be used to conveniently specify
-values in megabytes or gigabytes for example.
+values in megabytes, gigabytes or terabytes for example.
```
Comment (Simran-B) [inline 3.11/administration-configuration.md:195]:
```suggestion
values in megabytes, gigabytes, or terabytes for example.
```

File: 3.11/administration-configuration.md [hunk 2]
```diff
@@ -192,19 +192,21 @@ for small adjustments, you can put local overrides into a file
...
 |----------------------|----------|---------|
 | `kib`, `KiB`         | 1024     | 512KiB  |
 | `mib`, `MiB`         | 1024 ^ 2 | 64mib   |
 | `gib`, `GiB`         | 1024 ^ 3 | 3gib    |
+| `tib`, `TiB`         | 1024 ^ 4 | 3tib    |
 | `k`, `K`, `kb`, `KB` | 1000     | 3k      |
 | `m`, `M`, `mb`, `MB` | 1000 ^ 2 | 3mb     |
 | `g`, `G`, `gb`, `GB` | 1000 ^ 3 | 3GB     |
+| `t`, `T`, `tb`, `TB` | 1000 ^ 4 | 4TB     |
 | `%`                  | 0.01     | 5%      |
```
Comment (Simran-B) [inline 3.11/administration-configuration.md:207]: @nerpaula Should we change this table to use <code>base<sup>exp</sup></code> in the Factor column and add a **Bytes** column?

Comment (nerpaula) [reply 3.11/administration-configuration.md:207]: sounds good to me.

**Review by Simran-B — APPROVED** [review]

File: 3.11/administration-configuration.md [hunk 3]
```diff
@@ -190,21 +190,23 @@ for small adjustments, you can put local overrides into a file
...
+|----------------------|------------------|-------------------|-----------|
+| `kib`, `KiB`, `KIB`  | 1024             | 1,024             | `512KiB`  |
+| `mib`, `MiB`, `MIB`  | 1024<sup>2</sup> | 1,048,576         | `64mib`   |
+| `gib`, `GiB`, `GIB`  | 1024<sup>3</sup> | 1,073,741,824     | `3GIB`    |
+| `tib`, `TiB`, `TIB`  | 1024<sup>4</sup> | 1,099,511,627,776 | `3tib`    |
+| `k`, `K`, `kb`, `KB` | 1000             | 1,000             | `3k`      |
+| `m`, `M`, `mb`, `MB` | 1000<sup>2</sup> | 1,000,000         | `3M`      |
+| `g`, `G`, `gb`, `GB` | 1000<sup>3</sup> | 1,000,000,000     | `3GB`     |
+| `t`, `T`, `tb`, `TB` | 1000<sup>4</sup> | 1,000,000,000,000 | `4tb`     |
+| `%`                  | 0.01             |                   | `5%`      |
```
Comment (Simran-B) [inline 3.11/administration-configuration.md:207]:
```suggestion
| `%`                  | 0.01             | n/a               | `5%`      |
```

**Pull request merged**